python/sedona/maps/SedonaPyDeck.py

```diff
@@ -38,6 +41,14 @@ def create_choropleth_map(cls, df, fill_color=None, plot_col=None, initial_view_
         :param api_keys: Optional dictionary of API keys for Map providers
         :return: A pydeck Map object with choropleth layer added:
         """
+
+        for field in df.schema.fields:
+            if field.name == plot_col and field.dataType not in [IntegerType(), FloatType(), DoubleType(), LongType(),
+                                                                 DecimalType(), ShortType(), ByteType()]:
```

Comment:
Does ByteType work too?

Reply:
I don't see why it shouldn't. [As it is numeric. ](https://spark.apache.org/docs/latest/sql-ref-datatypes.html#:~:text=following%20data%20types%3A-,Numeric%20types,-ByteType%3A%20Represents%201)